builder/docker/driver_docker.go

```diff
@@ -57,6 +57,8 @@ func (d *DockerDriver) Build(args []string) (string, error) {
 		return "", fmt.Errorf("%s build failed: %s; stdout: %s; stderr: %s", d.Executable, err, stdout.String(), stderr.String())
 	}
 
```

Comment:
NIT: We can add a log line here that we are printing the Docker build output
```suggestion
log.Print("[DEBUG]" Logging Docker Build Output") 
```
or something similar, whatever you think would be feasible